processor/src/trace/tests/chiplets/hasher.rs

```diff
@@ -629,6 +631,19 @@ fn get_label_from_selectors(selectors: Selectors) -> Option<u8> {
     }
 }
 
+/// Populates the provided HasherState with the state stored in the decoder's execution trace at the
+/// specified row.
+fn fill_state_from_decoder_with_domain(
+    trace: &ExecutionTrace,
+    state: &mut HasherState,
+    row: usize,
+) {
+    let domain = extract_control_block_domain_from_trace(trace, row);
+    state[CAPACITY_DOMAIN_IDX] = domain;
+
+    fill_state_from_decoder(trace, state, row);
```

Comment:
In `fill_state_from_decoder()` we use `DECODER_HASHER_STATE_RANGE` constant which is defined locally. Since we defined a similar constant for `DECODER_OP_BITS_RANGE` in the `core` crate, should we move `DECODER_HASHER_STATE_RANGE` to core as well? Or alternatively define `DECODER_OP_BITS_RANGE` here?

Basically, these constants seem to be of the same type and to keep things consistent they should probably be defined in the same place.

Reply:
It was suggested that we would need `DECODER_OP_BITS_RANGE` in core when we get round to defining constraints, hence it was moved there to prevent this definition from being forgotten.  However, I've moved it back here as `DECODER_HASHER_STATE_RANGE` and `DECODER_OP_BITS_RANGE` are only used in this test file so it doesn't make sense for them to be in core at the minute.

Reply:
Yeah, I think it's fine for them to be in the test. My point was just that we don't want to end up with it being defined in multiple places later.

Reply:
Yeah I agree, it definitely opens up the scope for multiple definitions if we leave them here but I think it's the better of two evils in this instance.